packages/swatch/README.md

```diff
@@ -202,9 +202,9 @@ The `mixed-value` attribute and `mixedValue` property outline when an `<sp-swatc
 
 ```html
 <sp-swatch-group>
-    <sp-swatch mixed-value></sp-swatch>
-    <sp-swatch mixed-value rounding="full"></sp-swatch>
-    <sp-swatch mixed-value shape="rectangle"></sp-swatch>
+    <sp-swatch mixed-value aria-checked="mixed"></sp-swatch>
```

Comment:
I believe that westbrook touched on this before, but you shouldn't need to add aria-checked="mixed" here since it's doing it now (you would have to add selects="multiple" to the swatch group, though)? Or maybe you could say something like "Please note that the `aria-checked="mixed"` value only applies when the swatch is in a group with `selects="multiple"`.

I believe that even if you add it manually, like you have here, it still won't display on the screen reader because the role is still `button`.

Reply:
Thanks